hmda/src/test/scala/hmda/validation/rules/lar/validity/common/V714Spec.scala

```diff
@@ -12,10 +12,9 @@ class V714Spec extends LarEditCheckSpec {
   property(
     "If Application Channel exemption election is taken, all fields must be exempt") {
     forAll(larGen) { lar =>
-      lar.mustPass
 
-      lar.copy(applicationSubmission = ApplicationSubmissionExempt).mustFail
-      lar.copy(payableToInstitution = PayableToInstitutionExempt).mustFail
+      lar.copy(applicationSubmission = ApplicationSubmissionExempt, payableToInstitution = PayableToInstitutionNotApplicable).mustFail
```

Comment:
@KennethGudel should these test parameters be swapped? Application vs Payable?

Reply:
Yes, application `applicationSubmission` should go with `ApplicationSubmissionExempt` and `payableToInstitution` should go with `PayableToInstitutionExempt`. I'm not sure how these got swapped.

Reply:
@kgudel indeed they should

Reply:
indeed they should @kgudel